circe-new-day-notifier.el

```diff
@@ -63,16 +63,18 @@ documentation"
 ;;;###autoload
 (defun enable-circe-new-day-notifier ()
   (interactive)
-    (unless circe-new-day-notifier-timer
-      (setq circe-new-day-notifier-timer
-            (run-at-time "24:00:00" (* 24 60 60) 'circe-new-day-notification))))
+  (unless circe-new-day-notifier-timer
+    (add-to-list 'circe-format-not-tracked 'circe-new-day-notifier-format-message)
```

Comment:
Please use `push` in package code.

Reply:
Done.
For my own knowledge, is this recommended/explained somewhere?

Reply:
Yes, the `add-to-list` docstring says the following:

> This is handy to add some elements to configuration variables,
> but please do not abuse it in Elisp code, where you are usually
> better off using `push' or `cl-pushnew'.